lib/Graphs/SVFG.cpp

```diff
@@ -36,7 +36,7 @@
 #include "Util/Options.h"
 #include "MemoryModel/PointerAnalysisImpl.h"
 #include <fstream>
-
+#include "Util/Options.h"
```

Comment:
move these two headers to SVFGWriteRead.cpp

move `writeToFile readFile` to SVFGWriteRead.cpp